### PR TITLE
updated email server config 

### PR DESCRIPTION
**Pull request by mosabua**
including screenshot and adaptation of newer writing style, 

https://issues.sonatype.org/browse/NEXUS-9229

Also already implements change from https://issues.sonatype.org/browse/NEXUS-9291

Rendered output at http://bamboo.s/browse/BOOK-NEXUS46-1/artifact/STAGE/site/3.0/reference/admin.html#admin-system-emailserver

### CODE REVIEW

**Comment by mosabua** [comment]
The related config change for M6 was merged to master.

File: chapter-configuration.asciidoc [hunk 1]
```diff
@@ -185,36 +185,43 @@ backing up your configuration before proceeding.
...
-connection' button to confirm the configured parameters and the
-successful connection to the server. You will be asked to provide an
-email address that should receive a test email message. Successful
-sending will be confirmed in another pop up message.
+be encrypted.
+////
+
+Once you have configured the parameters you can use the 'Verify email server' button to confirm the configured
+parameters and the successful connection to the server. You are asked to provide an email address that should
+receive a test email message. Successful sending is confirmed in another pop up message.
```
Comment (joedragons) [inline chapter-configuration.asciidoc]: Minor: Suggest changing "another pop up" to "a" (a message).  It's a different type of popup so in this case I feel more detail could have misleading consequences.

**Comment by joedragons** [comment]
+1 with minor comment.  I also (minor) feel it's weird that you're merging this to NX3 "master" branch however the variable order change you made and updated screen for was in M6 and we have a M6 branch.  I can't comment on the image that I see, so making a note here.

**Comment by mosabua** [comment]
The re-ordered fields are minor compared to the simply wrong docs about SSL and so on live on the site. To minimize the need to update and at the same time be able to push this live asap I want to get this into the 3.0.x branch and publish. The m6 branch will probably get more changes on top of that once the SSL stuff gets implemented.

**Pull request merged**